pkg/rook/cr_status.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
How about naming the file `status.go`? It's always nice in go to have a single word filename if possible IMO.

Reply:
ack, done